### PR TITLE
docs: added `transfer-tokens` page

### PR DESCRIPTION
**Pull request by impelcrypto**
* feat: added `transfer-tokens` page 
  * Related: https://github.com/AstarNetwork/astar-docs/pull/74 
    * I've created new PR since I can't access to the branch above

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Would it be possible to add this to this PR? If so please do.

<meta name=“description” content=“Documentation of Astar and Shiden Network - The Future of Smart Contracts for Multichain - both WebAssembly (WASM) and EVM using XCM. Tutorials, user guides and tools for developers and builders.“>

**Comment by impelcrypto** [comment]
@Kahonnohak updated!

**Comment by Kahonnohak** [comment]
mmmm I think some layouts are now broken.

**Comment by Kahonnohak** [comment]
I don't know what is wrong but this link is not working. Please help. Thanks!
Screenshot 2022-09-07 at 11 48 53

**Comment by impelcrypto** [comment]
@Kahonnohak  Fixed.

> I don't know what is wrong but this link is not working. Please help. Thanks!

Because there was a space [here](https://github.com/AstarNetwork/astar-docs/pull/76/commits/413655798f738da5403ae8432c1dd04f077d3804#diff-bd68bd13a14f3362fb81f4841d35625713fd40b9ec5ec7df0a1f7531f1b5da60L96).

**Comment by Kahonnohak** [comment]
Thank you😭

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Review by andabakus — CHANGES_REQUESTED** [review]
There is one broken link (cannot understand why is not working.) 
maybe @impelcrypto will identify why?

the one in https://astar-docs--pr76-feat-added-new-trans-x5i7u5k2.web.app/docs/user-guides/setup-metamask

image

File: docs/user-guides/setup-metamask.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+sidebar_position: 3
+---
+
+# How to setup Astar Network to Metamask
+
+In this tutorial we will navigate you how to add Astar/Shiden Network to your Metamask account.
+
+<br />
+
+Adding Astar Network to Metamask is very easiy.
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply docs/user-guides/setup-metamask.md]: I've fixed in [this](https://github.com/AstarNetwork/astar-docs/pull/76/commits/562c2f61ed33d0d4c33636725ef2e6607bd98c24) commit.

File: docs/user-guides/setup-metamask.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<br />
+1. You will just need to go to [our Portal](https://portal.astar.network/) and try to connect to Metamask.
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply docs/user-guides/setup-metamask.md:14]: It works on my end. Please test it again.
https://gyazo.com/fe33f1900bb490f2067558cfbdd0fb31

[the remark this answers is not in the document]
Comment (andabakus) [reply docs/user-guides/setup-metamask.md:14]: Now it seems it works on my side as well 👍

**Review by andabakus — APPROVED** [review]

**Pull request merged**